How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md

```diff
@@ -1,16 +1,72 @@
+    * If the issue is more significant, flag as `bug`
+    * Other labels as appropriate
+
+    If there is a question as to the proper course of action on an issue, feel free to call @issueModerators to give an opinion.  Flag as `Discussing`.
+3. **Duplicate Issues**  
+If an issue is the same as another reported issue, the prior reported issue should take precedence.  Link to the other issue with #XXXX, where XXXX is the issue number and close the issue.
+4. **Bike Shedding**  
+Bike Shedding is an example of [Parkinson's law of triviality](https://en.wikipedia.org/wiki/Parkinson%27s_law_of_triviality).  Some issues are simply not worth fixing.  If you believe an issue is just a waste of time, flag as such and close.
+
+#### Pull Request Moderation
+Pull Requests (PRs) are how contributors to Free Code Camp submit changes to the repository for consideration.  It is important that these PRs are properly formatted and undergo thorough Quality Assurance Testing prior to being merged.
+
+##### PR Requirements and Formatting
+PRs must meet the following requirements:
+1. Must be against the `staging` branch
+2. Must be from a properly named non-staging branch on the user's fork
+3. Title must clearly identify the affected area/change made
+4. Title should NOT have an issue number in it
+5. Body of the PR should give details about the change as well as level of testing (IE: untested, tested locally)
+6. If the PR is in response to an open issue or issues, the body should also include `closes #XXXX` for each issue number closed
+7. Change should only have 1 commit
+8. Code should pass all tests and linting
+9. Code should be of general high quality and a needed change or improvement
+
+If the issue does not meet one or more of these requirements, note the deficiencies in a comment and/or highlight lines.  New Contributors may be referred to the HelpContributors Chat room.  At at the Mod's discretion the issue may be closed.
+
+##### Quality Assurance
+Assuming that the basic requirements for the PR are met, all PRs should undergo some level of Quality Assurance testing.  The most basic QA is to simply checkout the PR on a local copy of the site and test the changed functionality.  Be sure to read through the code changes and understand what the potential side effects are. Exercise code and corner cases.  Be sure to try both negative and positive test cases.
 
+If there is any doubt about the functionality, ask for the @issueModerators to also take look.
+
+For larger PRs, tag in @BerkeleyTrue.  In some cases it may make sense to test on the beta site.
+
+##### Special Requirements
+PRs which change the underlying function of the site or make non-trivial changes to the UI or UX of the site should be approved by @BerkeleyTrue or @QuincyLarson.  If you have any doubt, tag them in a comment and/or draw their attention to the PR via Gitter Chat.

```

Comment:
Maybe `@BerkeleyTrue` -> Berkeley's Gitter acc and `@QuincyLarson` -> Quincy's Gitter acc?